.github/workflows/gradle.yml

```
@@ -49,3 +47,19 @@ jobs:
+    - uses: actions/checkout@v4
+    - name: Set up openJDK version
+      uses: actions/setup-java@v4
+      with:
+        java-version: 21
+        distribution: 'zulu'
+    - name: Install Dependencies
+      run: npm install -g ro-crate-html-js
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v4
+    - name: Compile Javadoc
+      run: ./gradlew javadoc
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)